packages/ios_platform_images/example/ios/RunnerTests/UIImageIosPlatformImagesTests.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import ios_platform_images
+
+final class UIImageIosPlatformImagesTests: XCTestCase {
+
+  func testMultiResolutionImageUsesBest() {
+    if let image = flutterImageWithName(withName: "assets/multisize.png") {
+      XCTAssertNotNil(image)
```

Comment:
the `image` inside `if let` will always be non null. You should do something like: 

```
What if image is nil? should we fail the test instead? 

```
func testMultiResolutionImageUsesBest() {
  guard let image = flutterImageWithName(...) else {
    XCTFail("Image not found)
  }
  // continue here
}
```